packages/vscode-extension/project.json

```diff
@@ -46,7 +46,7 @@
 					"cp ./LICENSE dist/packages/vscode-extension",
 					"cp packages/vscode-extension/package.json dist/packages/vscode-extension",
 					"cp packages/vscode-extension/README.md dist/packages/vscode-extension",
-					"cp node_modules/@php-wasm/node/*.wasm dist/packages/vscode-extension"
+					"find node_modules/@php-wasm/node/ -type f -name php_*.wasm | while read FILE; do cp $FILE dist/packages/vscode-extension; done"
```

Comment:
Why update the vscode-extension code in this PR?

Reply:
@adamziel This was originally a companion PR to https://github.com/WordPress/wordpress-playground/pull/639, and when that project is built with sourcemaps, the WASM files appear in subpaths. This ensures the subpaths are copied over.

Reply:
Yeah it sounds relevant to #639, just not to pooling and request limiting this PR is dealing with. Would you please file an issue to keep track of this problem separately?